src/components/UNSTABLE__Pagination/Pagination.stories.js

```diff
@@ -92,7 +92,69 @@ storiesOf(components('UNSTABLE Pagination'), module)
     }
   )
   .add(
-    'with no page selector or sizer',
+    'with a page input',
+    () => (
+      <UNSTABLE__Pagination
+        {...props()}
+        totalItems={350}
+        pageSizes={array('Choices of `pageSize` (pageSizes)', [10, 20, 30])}
+      >
+        {({ currentPage, onSetPage, totalPages }) => (
+          <PageInput
+            currentPage={currentPage}
+            totalPages={totalPages}
+            id="number-input-1"
+            onChange={event => onSetPage(event.imaginaryTarget.value)}
```

Comment:
It's super important that people use `event.imaginaryTarget.value` instead of typical `event.target.value` because this component is controlled. 🤔 Do you think it is clear enough here & in the `info` for the story?